### PR TITLE
add mui and add a simple header with new routes

### PR DESCRIPTION
**Pull request by MaliyOna**

### CODE REVIEW

File: cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import PetsIcon from '@mui/icons-material/Pets';
+import Link from 'next/link';
+
+function HeaderWithMenu() {
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
```
Comment (v12code) [inline cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx]: Why don't you use just useState()

Comment (kalachck) [inline cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx]: Why don't you use just useState()?

File: cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+'use client'
```
Comment (kalachck) [inline cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx:1]: What is it?

**Review by kalachck — CHANGES_REQUESTED** [review]

File: cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx (hunk 2 above)
Comment (MaliyOna) [reply cat-note-client/app/shared/components/headerWithMenu/headerWithMenu.tsx:1]: image

**Review by v12code — APPROVED** [review]

**Pull request merged**